src/components/contextual/pages/vebal/LMVoting/GaugesTable.spec.ts

```diff
@@ -60,110 +60,73 @@ const global = {
   },
 };
 
+const gaugeId = '0x34f33CDaED8ba0E1CEECE80e5f4a73bcf234cfac';
 vi.mock('vue-router');
 vi.mock('@/providers/tokens.provider');
+vi.mock('@/composables/queries/useExpiredGaugesQuery');
 
-const gaugeId = '0x34f33CDaED8ba0E1CEECE80e5f4a73bcf234cfac';
-
-const pool: VotingPool = {
-  chain: GqlChain.Mainnet,
-  network: Network.MAINNET,
-  id: '0x1542b8783e5e884b6fe7422dd2f71a42c5edb86d0000000000000000000002f3',
-  address: '0x06Df3b2bbB68adc8B0e302443692037ED9f91b42',
-  type: GqlPoolMinimalType.Stable,
-  poolType: PoolType.Stable,
-  symbol: 'staBAL2',
-  tokens: [
-    {
-      address: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
-      weight: 'null',
-      symbol: 'USDC',
-      logoURI:
-        'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/logo.png',
-    },
-    {
-      address: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
-      weight: 'null',
-      symbol: 'USDT',
-      logoURI:
-        'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xdAC17F958D2ee523a2206206994597C13D831ec7/logo.png',
-    },
-  ],
+const pool: VotingPool = aVotingPool({
   gauge: {
     address: gaugeId,
     isKilled: false,
   },
-  votes: '8212341531532800',
-  votesNextPeriod: '6934407282299320',
-  userVotes: '0',
-  lastUserVoteTime: 0,
-};
+});
 
-const expiredGauges = [gaugeId];
 const votingPools: VotingPool[] = [pool];
 
 const queryExpiredLabel = () => screen.queryByText(/Expired/i);
-const queryVoteBtn = () => screen.queryByRole('button', { name: /Vote/i });
 const queryRemoveVotesBtn = () =>
   screen.queryByRole('button', { name: /Remove/i });
 
+const votingProviderPlugin = {
```

Comment:
I don't love this way of setting the providers but it's a workaround to avoid refactoring the whole testing setup.